blocks/top-table-list-block/features/top-table-list/_children/horizontal-overline-image-story-item.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Image /* , Video */ } from '@wpmedia/engine-theme-sdk';
+import { Image/* , Video */ } from '@wpmedia/engine-theme-sdk';
```

Comment:
this looks strange, is this commented out?

Reply:
see above